CONTRIBUTING.md

```diff
@@ -110,7 +110,7 @@ bundle exec maze-runner features/handled_errors.feature
 
 2. Checkout the `next` branch.
 
-3. Set the new version in `CHANGELOG.md`
+3. Set the new version in `CHANGELOG.md` and `Bugsnag/Assets/Bugsnag/Runtime/AssemblyInfo.cs`
```

Comment:
It might be worth adding a Rakefile command to do the version bump now there are >1 places to do this as there's a moderate chance of forgetting.

Suggest at least tasking up the idea if not something we add straightaway.